src/main/java/com/epam/lowcost/model/Flight.java

```diff
@@ -0,0 +1,80 @@
+package com.epam.lowcost.model;
+
+import java.math.BigInteger;
+import java.util.Date;
```

Comment:
Since Java 8 we have Date and Time API.  Please check it.

Reply:
Changed data types